airflow/api_fastapi/core_api/datamodels/connections.py

```diff
@@ -97,20 +102,23 @@ class ConnectionBulkCreateAction(BulkBaseAction):
 
     action: BulkAction = BulkAction.CREATE
     connections: list[ConnectionBody] = Field(..., description="A list of connections to be created.")
+    action_on_existence: BulkActionOnExistence = BulkActionOnExistence.FAIL
 
 
 class ConnectionBulkUpdateAction(BulkBaseAction):
     """Bulk Update Connection serializer for request bodies."""
 
     action: BulkAction = BulkAction.UPDATE
     connections: list[ConnectionBody] = Field(..., description="A list of connections to be updated.")
+    action_not_on_existence: BulkActionNotOnExistence = BulkActionNotOnExistence.FAIL
```

Comment:
Maybe:
```suggestion
    action_on_missing: BulkActionNotOnExistence = BulkActionNotOnExistence.FAIL
```
Or `action_on_non_existance` ?

Reply:
Sure, let's go with `action_on_non_existence`